runtime/near-vm-runner/RUNTIMES.md

```diff
@@ -0,0 +1,214 @@
+This document describes the considerations that go into picking a Wasm runtime as well as the
+conclusions we made after evaluating a number of VM backends and frontends.
+
+# Criteria
+
+There are a number criteria which have been used to evaluate a Wasm runtime for use in a NEAR
+validator. Some of the criteria are already listed in the [FAQ] document and here we take a more
```

Comment:
I don't think we should specify `validator` here. For this purpose, it should be the same for all nodes that validate some shard

Reply:
Reworded some occurences of validator to refer to the NEAR protocol instead, where makes sense.